### ISSUE TITLE
no metric requests_in_*

### ISSUE THREAD

**Issue by ihard** [go-graphite/carbonapi#303]
carbonapi 0.10.0.1
config:
```
listen: "0.0.0.0:8081"
concurency: 200
cache:
   type: "mem"
   size_mb: 2048
   defaultTimeoutSec: 60
   memcachedServers:
       - "127.0.0.1:1234"
       - "127.0.0.2:1235"
cpus: 10
tz: ""
sendGlobsAsIs: true
maxBatchSize: 100000000
graphite:
    host: "graphite:2003"
    interval: "60s"
    prefix: "dc.host.carbon-api.info"
    pattern: "{prefix}"

idleConnections: 50
pidFile: ""
upstreams:
    # Number of 100ms buckets to track request distribution in. Used to build
    # 'carbon.zipper.hostname.requests_in_0ms_to_100ms' metric and friends.
    # Requests beyond the last bucket are logged as slow
    # (default of 10 implies "slow" is >1 second).
    buckets: 10

    timeouts:
        global: "30s"
        afterStarted: "20s"
        connect: "200ms"

    concurrencyLimit: 0
    keepAliveInterval: "30s"
    maxIdleConnsPerHost: 100

    backends:
        - "http://127.0.0.1:9090"

    carbonsearch:
        backend: ""
        prefix: ""
    graphite09compat: false
expireDelaySec: 300
logger:
    - logger: ""
      file: "/var/log/carbonapi/carbonapi.log"
      level: "info"
      encoding: "json"
```
no graphite metric from section upstreams buckets:10 (example carbon.zipper.hostname.requests_in_0ms_to_100ms)

and not work metric (always =0)
zipper.timeouts
zipper.render_errors
zipper.find_errors

**Comment by azhiltsov** [comment]
all metrics for integrated zipper were moved to 
"carbon.api.%{host}.zipper." I believe.

**Comment by ihard** [comment]
I do not see them on my own or any other way.

**Comment by dcharleston** [comment]
I upgraded from 0.10.0.1 to 0.12.6 and don't see any of the carbon.api.<fqdn>.zipper metrics now. They were working before the upgrade.

**Comment by Civil** [comment]
@dcharleston your issue should be fixed in 0.13.0-rc.1 or higher.

However please do not comment on a bug that was about something that happened 2 years ago, unless you are really sure that it's exactly the same issue, as it likely would have different root cause and this comment would have very high chances to go unnoticed.
